libs/shared/ui/src/lib/components/icon/icons-status/unknow.tsx

```diff
@@ -0,0 +1,35 @@
+import { forwardRef } from 'react'
+import { twMerge } from 'tailwind-merge'
+import { type IconSVGProps } from '../icon'
+
+export const UnknowIcon = forwardRef<SVGSVGElement, IconSVGProps>(function UnknowIcon(
```

Comment:
`UnknowIcon` -> `UnknownIcon`
also rename the file itself

Reply:
nice catch